lib/index.js

```diff
@@ -25,7 +25,44 @@ module.exports = class {
    */
   issueRequest(method, url) {
     return this.axios({ method, url })
-      .then(({ data }) => data);
+      .then(({ data }) => data)
+      .catch(Promise.reject);
+  }
+  /**
+   * cid Request wrapper.
+   * @param  {String} method
+   *   Method for the HTTP request.
+   * @param  {String} url
+   *   The URL for the HTTP request.
+   * @return Mapped {Promises}
+   *   The response from the Drupal.org HTTP api.
+   */
+  cidRequest(method, url) {
+    return this.axios({ method, url })
+      .then(({ data }) => data)
+      .then(data => Promise
+        .all(data.list[0].comments
+          .map(e => this.comment({ cid: e.id })))
+              .then(x => x.map(e => e.list[0])))
+              .catch(Promise.reject);
+  }
+  /**
+   * tid Request wrapper.
+   * @param  {String} method
+   *   Method for the HTTP request.
+   * @param  {String} url
+   *   The URL for the HTTP request.
+   * @return Mapped {Promises}
+   *   The response from the Drupal.org HTTP api.
+   */
+  tidRequest(method, url) {
+    return this.axios({ method, url })
```

Comment:
We already have a request handler that should be used. Lets not add another reference to `axios` and all that comes with that.